packages/util/src/bytes.ts

```diff
@@ -430,6 +430,10 @@ export const bigIntToUnpaddedBytes = (value: bigint): Uint8Array => {
   return unpadBytes(bigIntToBytes(value))
 }
 
+export const bigIntToAddressBytes = (value: bigint): Uint8Array => {
+  return setLengthLeft(bigIntToBytes(value), 20).slice(-20)
+}
```

Comment:
Seems to me such a new method might be reasonable and justified. If we want to explicitly add we would need 2-3 explicit test cases though. Also I guess this should (?) throw if the big int value exceeds the 20 bytes.

Reply:
(less-work-alternative: do not add new method but use code from method directly)

Reply:
good idea to add some test cases :+1: .

not so sure about throwing because we were already doing slice(-20) in the evm code where i also replaced this method. I mean would we error in the EVM execution if the stack input of the address would > 20 bytes?

Reply:
I guess that's a question for @jochem-brouwer 🙂

Reply:
Good point, I actually think that EVM does **not** throw here, especially because it will throw in the new EOF cases in case that you put more than 20 bytes on the stack. (So, in current CALLs we slice the stack with 20 bytes, but in EOF if the stack has more than 20 bytes it throws).

This should likely fail some ethereum/tests cases in case where we start throwing on these CALL situations.

Reply:
we can add an extra flag : `strict: boolean` , if passed true (for e.g. EOF activated), then we can throw else do the current behavior

Reply:
Can we do it other way around and have the strict case as the default and so set the flag to true and only deviate (to the old behavior) if false is passed? (so this obviously would only work for the new method due to backwards compatibility needs of existing Util methods)

Reply:
sounds good, will impl that way :+1:

Reply:
modified :+1: